2024 Preperation

Abban opened the pull request:
This is a parent ticket for general 2024 clean up PRs

moiikana commented on desktop/styles/styles_var.pcss:
```diff
@@ -2,12 +2,18 @@
+@import './Tooltip.pcss';
 @import '../../shared/components/ui/FundsModal.pcss';
+
+.wmde-banner {
+	p {
+		margin: 0 0 8px;
+	}
+}
```
the placing of this is a little hidden here .. even if it's used at many places, maybe Banner.pcss or sth would be a better place.

Abban committed:
Remove archived banners

Abban committed:
Fix up live date and time

- Make the current date and time content return an object
- Remove unused methods and objects
- Rename and update composable
- Update tests

Abban committed:
Fix up RuntimeEnvironment injection

- Make the constructor parameters in the impression count
  and trackers required
- Remove lazy loading
- Update tests
- Fix banners that were missing the explicit setup

Abban committed:
Merge pull request #350 from wmde/live-date-time

Fix up live date and time

Abban committed:
Merge pull request #351 from wmde/runtime-environment

Fix up RuntimeEnvironment injection

Abban committed:
Update content
